app/templates/views/accessibility_statement.html

```diff
@@ -166,10 +162,11 @@ <h2 class="heading-medium" id="how-we-tested-this-service">
   </h2>
 
   <p class="govuk-body">
-    The parts of the website used to send text messages, emails and letters were last tested in February 2021. The emails we send out, the web pages users see when they are sent a file by Notify and a sample letter PDF were last tested on 11 March 2019. Both tests were carried out by the Digital Accessibility Centre.
+    The parts of the website we added to let users create new email and letter designs were tested on 9 October 2023. The button for copying text in the page to your clipboard and the component for choosing the day and time to send notifications were also tested as part of this.
```

Comment:
```suggestion
    The parts of the website we added to let users create new email and letter designs were tested on 9 October 2023. The button for copying text in the page to your clipboard and the component for choosing the day and time to send notifications were also tested as part of this.
```

Is this referring to branding?

Reply:
Yes. I was wary of using the term as it seems part of our internal language but also unsure if 'design' is a good substitute. And very much up for suggestions :)

Reply:
We use ‘email branding’ and ‘letter branding’ in user-facing content, so we can probably say that here too.

We can always link to the guidance for anyone who wants to understand more.

```suggestion
    The parts of the website used to add <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_email_branding') }}">email branding</a> or <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_letter_branding') }}">letter branding</a> were tested on 9 October 2023. The button for copying text in the page to your clipboard and the component for choosing the day and time to send notifications were also tested as part of this.
```